core/src/main/java/tc/oc/pgm/blitz/BlitzModule.java

```diff
@@ -47,6 +47,9 @@ public BlitzModule parse(MapFactory factory, Logger logger, Document doc)
       for (Element blitzEl : blitzElements) {
         boolean broadcastLives = XMLUtils.parseBoolean(blitzEl.getChild("broadcastLives"), true);
         int lives = XMLUtils.parseNumber(Node.fromChildOrAttr(blitzEl, "lives"), Integer.class, 1);
```

Comment:
Instead of adding explicit checks, you can change the signature of this instance of `parseNumber` and provide a `Range`. See `XMLUtils#parseNumber`